### PR TITLE
[ODS-5494] Expand coverage for disabling PGBouncer in favor of client-side pooling

### PR DESCRIPTION
**Pull request by gmcelhanon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,19 +37,27 @@ pb-ods:
...
+
+However, to remove PGBouncer as a dependency and configure client-side connection pooling instead, make the following changes:
 
 1. Remove and/or comment out the `pb-admin` and `pb-ods*` services from the compose file.
 2. Update `admin`, and `api` services and change the database environment variables (`ODS_POSTGRES_HOST`, and `ADMIN_POSTGRES_HOST`) to point to `db-ods` and `db-admin` from `pb-ods` and `pb-admin`.
 3. Update the `POSTGRES_PORT` environment variable to point to 5432.
 4. If required expose the port on the services `db-admin`, and `db-ods` services. Note these ports should be independent.
+5. Apply connection string settings for [client-side connection pooling](https://www.npgsql.org/doc/connection-string-parameters.html?q=pooling#pooling) in the _appsettings.template.json_ files for the applicable containers.
```
Comment (vimayya) [inline README.md:58]: While modifying appsettings.template.json, official images can't be used, is there a way to pass in as env variable but have a separate pool size for admin app and api?

Comment (gmcelhanon) [reply README.md:58]: Yes, and I have a branch that I started when working on this ticket with exactly that sort of approach, which we'd need to continue development on. We'd just need separate environment variables for the Admin's ODS pool size and the ODS API's ODS pool size.

File: .env.example [hunk 2]
```diff
@@ -30,6 +30,20 @@ POSTGRES_PASSWORD=<password for default postgres user>
 # The following configuration variable PGBOUNCER_QUIET="true" which will suppress this messaging.
 PGBOUNCER_QUIET="true"
 
+# PostgreSQL client-side pooling (see repository README file for additional information)
```
Comment (vimayya) [inline .env.example:33]:
```suggestion
# PostgreSQL client-side pooling. Consider only if not using PgBouncer (see repository README file for additional information)
```

**Comment by vimayya** [comment]
@gmcelhanon Code changes and readme looks good to me. I haven't done any testing.

**Review by AxelMarquez — CHANGES_REQUESTED** [review]
The containers don't pick up the new env var values set up in the .env file since the compose files don't forward them yet. For example, if I set `NPG_POOLING_ENABLED=true` in the .env file it doesn't get applied in the container.

To do so, add each env var to the environment section in the image definition in the compose files.

**Review by AxelMarquez — APPROVED** [review]

**Pull request merged**